### ISSUE TITLE
USER ISSUE: main menu not showing (0.10.0.0 beta playtest-12)

### ISSUE THREAD

**Issue by AriePelt** [StrangeLoopGames/EcoIssues#23867]
when opening the game I get no menu to join or start a server. 
![image](https://github.com/StrangeLoopGames/EcoIssues/assets/77667688/d5f8a713-c078-459d-a602-b20fdd7e4435)

Version: 0.10.0.0 beta playtest-12

Steps to Reproduce:
- Select wtplaytest (or staging) branch under steam
- Open game 

Opening the game when not having a steam beta selected does work as expected and allows me to join a server. (-prev log is from changing branch to main and joining a server from there)

[Player-prev.log](https://github.com/StrangeLoopGames/EcoIssues/files/11898352/Player-prev.log)
[Player.log](https://github.com/StrangeLoopGames/EcoIssues/files/11898353/Player.log)

**Comment by SlayksWood** [comment]
@AriePelt Hi, thank you for the report, do you still have the issue, can you try to start game with Eco.exe(not in steam) and log in. Will you still have the issue?

We had such issue a couple monthes ago when first time start game in 10.0, and I hoped it was fully fixed. I can't reproduce it for now from my side.

**Comment by AriePelt** [comment]
Well don't know if latest update worked or it was just waiting a bit, but now it does work through steam.

Just for additional information I did restart steam/my pc/reinstalled the game (with removing all files) yesterday. I also did try connect to WT directly through the quickjoin link on the webUI. That asked for username and password, but got a SSL error (through steam didn't give a in game error). Looked through the player log and looked the same as connecting through steam (didn't save that log though).

If there is any other information/data you want from me that might help with this in the future let me know and I will collect it, but the issue can probably be closed as it does work now
